app/scripts/controllers/transactions/tx-state-manager.js

```diff
@@ -361,13 +361,15 @@ class TransactionStateManager extends EventEmitter {
     @param err {erroObject} - error object
   */
   setTxStatusFailed (txId, err) {
+    let error = !err ? Error('Internal metamask failure') : err
```

Comment:
shouldn't be `new Error('Internal metamask failure')` ?

Reply:
Also the linter is failing because it wants a const instead of let

Reply:
ooh woops